### ISSUE TITLE
Blade echo breaks if it echos a string containing the phrase ' or ' in a ternary result

### ISSUE THREAD

**Issue by LiddleDev** [laravel/framework#24238]
- Laravel Version: 5.6.15
- PHP Version: 7.2.0
- Database Driver & Version:

### Description:
When using {{ }} to echo with a ternary inside, it will break if one of the two resulting strings contain the phrase ' or '

You will receive the error
```
Parse error: syntax error, unexpected '' : '' (T_CONSTANT_ENCAPSED_STRING), expecting ',' or ')'
```

### Steps To Reproduce:
* Create a variable $a and set it to true.
* Add the following to a blade file

```
{{ $a ? ' or something' : '' }}
```

**Comment by staudenmeir** [comment]
This is a known issue: https://github.com/laravel/framework/issues/23528

**Comment by LiddleDev** [comment]
Ah thanks, fixed in 5.7

https://github.com/laravel/framework/pull/23532
